### DISCUSSION TITLE
ramdisk vs 2TB or larger grade "A" consumer nvme comparison?

### DISCUSSION THREAD

**Discussion by sigi77777** [madMAx43v3r/chia-plotter#98] [General]
Did anyone compare performance on the same machine? Wonder how faster would it be with ramdisk setup to temp2 dir.

**Comment by madMAx43v3r** [comment]
Depends on your CPU power, if CPU is already a bottleneck without ramdisk, then it won't help much, except reduce SSD wear.

**Reply by sigi77777** [reply]
What is the proper way of finding the bottlneck?
in ssd i suppose IO wait que length, but at cpu?
